### PR TITLE
feat: add "tabs" component

### PR DESCRIPTION
**Pull request by leMaur**
## Summary
https://app.clickup.com/t/rg9fnq

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [x] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
@leMaur can you use it on one of the projects so it can be seen in action too?

File: resources/views/tabs/tab.blade.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<button
+    type="button"
+    class="tab-item transition-default"
+    :class="{ 'tab-item-current': selected === '{{ $name }}' }"
+    @click="select('{{ $name }}')"
+    @keydown.enter="select('{{ $name }}')"
+    @keydown.space.prevent="select('{{ $name }}')"
+    role="tab"
+    id="tab-{{ $name }}"
+    {{--aria-controls="panel-{{ $name }}"--}}
```
Comment (ItsANameToo) [inline resources/views/tabs/tab.blade.php:12]: ?

File: EXAMPLES.md [hunk 2]
```diff
@@ -763,3 +763,44 @@ mix.extract(['chart.js']);
...
 | theme | The theme to apply. (an array with 'name' and 'mode'. E.g. ['name' => 'grey', 'mode' => 'light']) | no | |
+
+
+### Tabs
+
+Add the following to the `app.js` file:
+
+```js
+import "@ui/js/tabs.js";
+```
```
Comment (ItsANameToo) [inline EXAMPLES.md:774]: add this to the readme as that contains the setup for the other components that require specific actions too

**Comment by ItsANameToo** [comment]
some remarks on the related nodem PR that will apply here

File: EXAMPLES.md (hunk 2 above)
Comment (leMaur) [reply EXAMPLES.md:774]: @ItsANameToo Do I have to add this on README.md or add a notes in the EXAMPLES.md (see below)?

```
> Be sure to follow the instruction on [README.md](/README.md#installation)
```

Comment (ItsANameToo) [reply EXAMPLES.md:774]: Add the setup to the readme with a quick example on how it should be used (see how it's done for other components). Then the full example and properties overview can be in the example file

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**